### DISCUSSION TITLE
Can't performance actions ... while the node is passive

### DISCUSSION THREAD

**Discussion by famschopman** [ravendb/ravendb#13789] [Q&A]
Probably a noob thing. Just upgraded to latest stable, copied my RavenData folder, renewed the license and then tried to access the databases. Unfortunately she is not playing along. 

When looking under the server dashboard, everything is fine with two databases in the green.

How can I make the node less passive?

<img width="978" alt="image" src="https://user-images.githubusercontent.com/16013305/157679745-74a8fa2e-8f6f-4974-bc1e-1f717ded96bf.png">

**Comment by ayende** [comment]
Did you delete the `System` database or removed this from a cluster?
You can do the the Manage Server -> Cluster tab and click on Make Active.

**Comment by famschopman** [comment]
I reproduced the issue. What I initially did

a) Download RavenDB-5.3.100-windows-x64, extract to C:\RavenDB-5.3.100-windows-x64 
b) Copy the entire RavenData folder from RavenDB-4.2.4-windows-x64 (incl. Databases and System folder)
c) Then do the first time startup of Raven.Server, accept statement, provide port number and so on. 

This one failed. I just did the following, so only copy the RavenData folder after starting up Raven for the first time.

a) Extract to C:\RavenDB-5.3.100-windows-x64 
b) Then do the first time startup of Raven.Server, accept statement, provide port number and so on. 
c) Copy the entire RavenData folder from RavenDB-4.2.4-windows-x64 (incl. Databases and System folder)

This works. Databases can be accessed now.

**Reply by ayende** [reply]
Okay, you are doing double initialization here, which make more sense. Basically, you have a mismatch between the `System` database and the `settings.json`, which cause us to start in a "new node" mode.

The way to handle that properly is to copy both the `settings.json` and the `System` / `RavenData` directly.
Also, any certificates, etc.

**Reply by garayx** [reply]
Another option would be to extract the RavenDB-5.3.100-windows-x64 files into RavenDB-4.2.4-windows-x64 folder, which will overwrite the RavenDB binaries, and keep your RavenData, settings.json and certificate if there is any.

**Reply by famschopman** [reply]
> Another option would be to extract the RavenDB-5.3.100-windows-x64 files into RavenDB-4.2.4-windows-x64 folder, which will overwrite the RavenDB binaries, and keep your RavenData, settings.json and certificate if there is any.

There are alot of various subfolders in the /Server/ folder including resources and other libraries so I took the safe way and just copied the entire thing. 

It is more a reminder for the future that I should not copy the RavenData folder before your first cold boot of RavenDB Server.

Now that I know, I remember it for the next time.
